### DISCUSSION TITLE
Review assign error

### DISCUSSION THREAD

**Discussion by aaimtiaz** [AstroBridge/BDLensing#177] [General]
After creating a pull request, there's been an error message "GitHub API error (message: Request failed with status code 403). "github-token" may not be correct." What should I do?
![Screenshot 2023-07-07 103616](https://github.com/ajshajib/BDLensing/assets/78560474/edb5d300-59fc-436e-91c4-d2a57d44573f)

**Comment by ajshajib** [comment]
It's taken care of for now, not sure if it's completely fixed, though, so it may happen for a future PR. I will try to find another solution in that case.

**Comment by aaimtiaz** [comment]
This error message has also been seen in pull request, #178

**Reply by ajshajib** [reply]
This issue should be fixed now for future PRs, according to my test. 🤞

**Comment by Rajit13** [comment]
I am having this problem for quite some time now
![image](https://github.com/ajshajib/BDLensing/assets/81336262/744c78e9-ad63-40f6-9a80-dbfc5f667096)
![image](https://github.com/ajshajib/BDLensing/assets/81336262/93055965-c838-44cd-9c31-86fac7a2e793)

**Reply by aaimtiaz** [reply]
Haven't faced any issue with the ReviewNB, working smoothly.

**Reply by SulymanHossain** [reply]
> I am having this problem for quite some time now
> ![image](https://github.com/ajshajib/BDLensing/assets/81336262/744c78e9-ad63-40f6-9a80-dbfc5f667096)
> ![image](https://github.com/ajshajib/BDLensing/assets/81336262/93055965-c838-44cd-9c31-86fac7a2e793)
> 

I'm also facing this issue.

**Reply by ajshajib** [reply]
@aaimtiaz Does your GitHub account have the education plan? Just trying to figure out the difference between your account and those having the above issue.

**Reply by aaimtiaz** [reply]
> @aaimtiaz Does your GitHub account have the education plan? Just trying to figure out the difference between your account and those having the above issue.

Maybe not. Though I am not quite sure what the education plan means in this case.

**Comment by aaimtiaz** [comment]
> I am having this problem for quite some time now ![image](https://private-user-images.githubusercontent.com/81336262/252303034-744c78e9-ad63-40f6-9a80-dbfc5f667096.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE2OTAxNTI3MjksIm5iZiI6MTY5MDE1MjQyOSwicGF0aCI6Ii84MTMzNjI2Mi8yNTIzMDMwMzQtNzQ0Yzc4ZTktYWQ2My00MGY2LTlhODAtZGJmYzVmNjY3MDk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzA3MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMwNzIzVDIyNDcwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5MjNmNDkwY2IxZWU5YWRjN2Q1ZTc3ZTVjMDYxMjMwYmFmMWE1Y2MxMjhlZGU4ZjkxMjY3Mzk5OTI1MDk5ZjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lnfDMhbHTdYayc_eTHNxNF0gxYEoAYPDr_3LYb6keuQ) ![image](https://private-user-images.githubusercontent.com/81336262/252303153-93055965-c838-44cd-9c31-86fac7a2e793.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE2OTAxNTI3MjksIm5iZiI6MTY5MDE1MjQyOSwicGF0aCI6Ii84MTMzNjI2Mi8yNTIzMDMxNTMtOTMwNTU5NjUtYzgzOC00NGNkLTljMzEtODZmYWM3YTJlNzkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzA3MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMwNzIzVDIyNDcwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRiMGQ3MjY0MzdiNmRhOTc1NTQ2Y2E3NjZmY2Q5NTY2M2RlNWM4Y2JhMmNmMmNlMzNlMzExZDJlZTJhMDMzZDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.P8Htcr0fZs0OANH9aNciTJ7KNENPjTKS9VOszA4wQtE)

Now I am facing similar problem.
